src/MBrace.Flow/CloudFlow.fs

```diff
@@ -1123,3 +1139,55 @@ module CloudFlow =
                         return! projection collector.Result
                 } }
 
+
+
+
+                
+    /// <summary>Creates a CloudFlow from the ReceivePort of a CloudChannel</summary>
+    /// <param name="channel">the ReceivePort of a CloudChannel.</param>
+    /// <returns>The result CloudFlow.</returns>
+    let ofCloudChannel (channel : IReceivePort<'T>) : CloudFlow<'T> =
```

Comment:
Maybe for this PR it is better to keep a clear seperation between the concepts of Channel(Send/Receive)Port and the new abstraction Cloud(Source/Sink). Cloud(Source/Sink) are more general in nature and a simple renaming is a bit premature.